pkgs/by-name/qu/quill-log/package.nix

```
@@ -0,0 +1,25 @@
+{ lib, stdenv, fetchFromGitHub, cmake }:
+
+stdenv.mkDerivation rec {
+  pname = "quill-log";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    owner = "odygrd";
+    repo = "quill";
+    rev = version;
+    sha256 = "c97bf3bfac6dfb7ed77fa08d945a490e302ba07e405539fda61985b39750cb29";
```

```suggestion
    hash = "sha256-yXvzv6xt+37Xf6CNlFpJDjAroH5AVTn9phmFs5dQyyk=";
```
same hash, just different format